src/controllers/expense.controller.js

```diff
@@ -0,0 +1,132 @@
+const expensesHelpers = require('../helpers/expense.helpers');
+const expensesService = require('../services/expense.service');
+const {
+  INTERNAL_SERVER_ERROR,
+  EXPENSE_NOT_FOUND_ERROR,
+} = require('../utils/config');
+
+const create = async (req, res) => {
+  try {
+    const {
+      userId,
+      spentAt,
+      title,
+      amount,
+      note,
+      category = 'Other',
+    } = req.body;
+
+    const requestData = {
+      userId,
+      title,
+      amount,
+      category,
+      note,
+      res,
+    };
+
+    if (
+      expensesHelpers.isUserExist(userId, res) ||
+      expensesHelpers.validateRequestData(requestData)
+    ) {
+      return;
+    }
+
+    const expense = await expensesService.create({
+      userId,
+      title,
+      spentAt,
+      category,
+      amount,
+      note,
+    });
+
+    res.status(201).send(expense);
+  } catch (error) {
+    res.status(500).send(INTERNAL_SERVER_ERROR);
+  }
+};
+
+const get = async (req, res) => {
+  try {
+    const { userId, categories, from, to } = req.query;
+
+    const expenses = await expensesService.getExpenses({
+      userId,
+      categories,
+      from,
+      to,
+    });
+
+    res.send(expenses);
+  } catch (error) {
+    res.status(500).send(INTERNAL_SERVER_ERROR);
+  }
+};
+
+const getOne = async (req, res) => {
+  try {
+    const { id } = req.params;
+
+    const expense = await expensesService.getExpenseById(id);
+
+    if (!expense) {
+      return res.status(404).send({ EXPENSE_NOT_FOUND_ERROR });
```

Comment:
In this case on the client you see the object where the key and value are the same, probably it should be different. 
```
{
  'Expense not found': 'Expense not found',
}
```
Also in some cases, I see you send just string, standardized your error responses